### PR TITLE
Implemented 4 filters

### PR DESCRIPTION
**Pull request by axfcampos**
Implemented 4 filters: withoutLabel, noOneAssigned, noMilestone, noComments

### CODE REVIEW

File: resources/index.js [hunk 1]
```diff
@@ -1 +1,7 @@
-exports.untouched = require('./untouched.js');
\ No newline at end of file
+exports.untouched = require('./untouched.js');
+exports.withoutLabel = require('./withoutLabel.js');
+exports.noOneAssigned = require('./noOneAssigned.js');
+exports.noMilestone = require('./noMilestone.js');
+exports.noComments = require('./noComments.js');
+exports.noActivity7days = require('./noActivity7days.js');
```
Comment (daviddias) [inline resources/index.js]: Please don't use numbers for name, I know JS allows it, but should mean we have to use everything it does :)

File: resources/untouched.js [hunk 2]
```diff
@@ -20,10 +20,10 @@ function create(request, reply) {
       a: 'active'
     };
     reply.view('template', context);
-    // console.log(filtered[1]);
+    console.log(filtered[0]);
```
Comment (daviddias) [inline resources/untouched.js]: This is just debug, let it commented out to avoid log spam

**Comment by daviddias** [comment]
Just minor stuff to correct, great work! Thank you!

**Comment by axfcampos** [comment]
Implemented the other 2 filters, all 6 are working now

In case the issue has no comments it'll pick the original create date to decide if its older than year or seven days. 

Also counting years as 365 days for now, could implement leap year awareness in the future.

ˆ_ˆ

**Comment by daviddias** [comment]
Thanks @axfcampos !! :D 

Having leap years will definitely be important since there are issues with two years or more without activity.

Going to merge this today

File: modules/fetchIssues.js [hunk 3]
```diff
@@ -63,61 +64,103 @@ function storeIssue(issue) {
...
-}
\ No newline at end of file
+  if(comments.length === 2){ return null; }
+
+  JSON.parse(comments).forEach(function (comment) {
+
+    //change from ISO time format to date
+    //console.log(comment.updated_at)
+    var time = new Date(comment.updated_at);
+    time =   new Date( time.getTime() + ( time.getTimezoneOffset() * 60000 ) ).getTime();
```
Comment (daviddias) [inline modules/fetchIssues.js]: @axfcampos this seems weird, why are you sure that was `time` var that you wanted to use for the second time?

Also, the last date of activity should live in a different module

File: modules/fetchIssues.js [hunk 4]
```diff
@@ -63,61 +64,103 @@ function storeIssue(issue) {
...
+          cleanIssue.body = issue.body;
+          cleanIssue.user = issue.user;
+          cleanIssue.labels = issue.labels;
+          cleanIssue.assignee = issue.assignee;
+          cleanIssue.milestone = issue.milestone;
+          cleanIssue.comments = issue.comments;
+          cleanIssue.pullRequest = issue.pull_request;
+          cleanIssue.closedAt = issue.closed_at;
+          cleanIssue.createdAt = issue.created_at;
+          cleanIssue.updatedAt = issue.updatedAt;
```
Comment (daviddias) [inline modules/fetchIssues.js]: Doesn't this date represent the last activity in a issue??

It seems so by the API docs
http://developer.github.com/v3/issues/

**Comment by axfcampos** [comment]
Although the date was returning from the sever and being saved the right way in the db, after being queried it came with the current time date.

After cleaning up all the db files this stopped happening and worked well. It appears the updatedAt date field in the db wasn't updating.

**Pull request merged**